CHANGELOG.md

```diff
@@ -20,6 +20,29 @@ Code v99.99.999
 
 -->
 
+## [4.8.0](https://github.com/coder/code-server/releases/tag/v4.8.0) - 2022-10-24
+
+Code v1.72.1
+
+### Added
+
+- Support for the Ports panel which leverages code-server's built-in proxy. It
+  also uses `VSCODE_PROXY_URI` where `{{port}}` is replace when forwarding a port.
+  Example: `VSCODE_PROXY_URI=https://{{port}}.kyle.dev` would forward an
+  application running on localhost:3000 to https://3000.kyle.dev
```

Comment:
Love the example!  Wonderful use of the changelog.

Reply:
"Don’t let your friends dump git logs into changelogs." - applying what I learned from you!